components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/context/SessionContext.java

```diff
@@ -37,6 +34,7 @@
     private boolean isRememberMe = false;
     private Map<String,Object> properties = new HashMap<>();
     private SessionAuthHistory sessionAuthHistory = new SessionAuthHistory();
+    private Map<String, Map<String, AuthenticatedIdPData>> authenticatedIdPsPerApp = new HashMap<>();
```

Comment:
Good to have a comment implying what each dimensional key refer to. One is sessionID and the other is application name?

Reply:
fixed